File: Swift/NaturalNumbers.swift
print ( "Up to which number you want to print natural numbers", terminator : "" )
var Count = Int (readLine()!)!
var Counter = 1
print ( "The first \(Count) natural numbers are", terminator: "")
while ( Counter <= Count ){
  print(" \(Counter)", terminator: "")
  Counter = Counter + 1
}
print (".")

File: Python/NaturalNumbers.py
Number = int ( input ( 'Up to which number you want to print natural numbers:' ) )
print ('The first '+ str(Number) + ' natural numbers are')
for Counter in range ( 0, Number, 1 ):
  print (' ' + str ( Counter + 1 ) ,)
print ( '.' )
  